src/stylesheets/roleAssignmentPanel.scss

```diff
@@ -0,0 +1,42 @@
+$components-width: 14rem;
+
+.roleAssignmentPanel {
```

Comment:
If you want to follow BEM convention ( I guess so) its good to replace camelCase with dash-separator-convention

http://getbem.com/naming/